### PR TITLE
Watson Simulation

### PR DESCRIPTION
**Pull request by Jorgelduarte**

### CODE REVIEW

File: Untitled-1 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+curl -X POST \
+-H "Content-Type: application/json" \
+-u "ac49cf55-8ee6-485e-aed0-956429714dc0":"Rfeif6hCMbLD" \
+-d @parameters.json "https://gateway.watsonplatform.net/natural-language-understanding/api/v1/analyze?version=2018-03-16"
```
Comment (tinanwlin) [inline Untitled-1]: Seems like this is a test file, we may need to remove this file

File: app/controllers/watson_controller.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        begin
+        
+            
+            url = URI("https://gateway.watsonplatform.net/natural-language-understanding/api/v1/analyze?version=2018-03-16")
+            
+            http = Net::HTTP.new(url.host, url.port)
+            http.use_ssl = true
+            http.verify_mode = OpenSSL::SSL::VERIFY_NONE
+            
+            req = Net::HTTP::Put.new(url)
```
Comment (tinanwlin) [inline app/controllers/watson_controller.rb]: Is it a Put or Post request?

File: parameters.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+      "sentiment": false,
+      "limit": 2
+    },
+    "keywords": {
+      "emotion": true,
+      "sentiment": false,
+      "limit": 2
+    }
+  }
+}
```
Comment (tinanwlin) [inline parameters.json]: This file come along with the Untitled-1 file, should we delete it as well?

File: Untitled-1 (hunk 1 above)
Comment (Jorgelduarte) [reply Untitled-1]: Tina, maybe that file could be useful on monday

File: app/controllers/watson_controller.rb (hunk 2 above)
Comment (Jorgelduarte) [reply app/controllers/watson_controller.rb]: I have no idea. I tried a lot of codes (you can see it in the same file at the bottom another test) Maybe someone could help me on Monday.

File: parameters.json (hunk 3 above)
Comment (Jorgelduarte) [reply parameters.json]: As the tutorial from IBM-Watson, they put separately. The same situation Untitled-1.

**Pull request merged**